analysis/rogueoutlaw/src/modules/features/RollTheBonesCastTracker.js

```diff
@@ -12,14 +12,14 @@ export const ROLL_THE_BONES_CATEGORIES = {
   HIGH_VALUE: 'high',
 };
 
-let BROADSIDE_VALUE = { sleight_of_hand: 2, base: 3 };
-let TRUE_BEARING_VALUE = { sleight_of_hand: 2, base: 3 };
-let RUTHLESS_PRECISION_VALUE = { sleight_of_hand: 2, base: 2 };
-let SKULL_AND_CROSSBONES_VALUE = { sleight_of_hand: 2, base: 2 };
-let BURIED_TREASURE_VALUE = { sleight_of_hand: 1, base: 1 };
-let GRAND_MELEE_VALUE = { sleight_of_hand: 1, base: 1 };
-
-var BUFF_VALUE_BY_ID = {};
+const BROADSIDE_VALUE = { sleight_of_hand: 2, base: 3 };
+const TRUE_BEARING_VALUE = { sleight_of_hand: 2, base: 3 };
+const RUTHLESS_PRECISION_VALUE = { sleight_of_hand: 2, base: 2 };
+const SKULL_AND_CROSSBONES_VALUE = { sleight_of_hand: 2, base: 2 };
+const BURIED_TREASURE_VALUE = { sleight_of_hand: 1, base: 1 };
+const GRAND_MELEE_VALUE = { sleight_of_hand: 1, base: 1 };
+
+let BUFF_VALUE_BY_ID = {};
```

Comment:
fun fact: assigning props to an object works on `const` objects. `const` only controls whether you can use `=`. for example:

```js
const a = {};
a.foo = 1;
```
works but
```js
const a = null;
a = 1;
```
does not.

not gonna block the review over it but its good to know